.semaphore/semaphore.yml

```diff
@@ -17,6 +17,7 @@ blocks:
           - checkout
           - cache restore client-node-modules-$SEMAPHORE_GIT_BRANCH-$(checksum yarn.lock),client-node-modules-$SEMAPHORE_GIT_BRANCH,client-node-modules-master
           - nvm install
+          - nvm use
```

Comment:
@allison-c It looks like Balint also applied the [same fix](https://github.com/acl-services/paprika/pull/1306), but left in `nvm use` too.  Maybe it's redundant? Probably doesn't hurt anything though?

Reply:
yeah doesn't hurt anything, ``nvm install` will implicitly use the newly-installed version` , so not necessary though

Reply:
The unnecessary `nvm use` has been removed, thank you for the heads up!